source/server/getting-started/launch-script.rst

```diff
@@ -30,6 +30,26 @@ Windows
 
 Save your Windows launch script as ``launch.bat``.
 
+.. warning::
+   Launching your server in a Command Prompt or PowerShell environment can lead to the server randomly hanging. During 
```

Comment:
it doesn't randomly hang, it happens when a user marks something

Reply:
Removed randomly. Waiting on more feedback before pushing a commit.